native/src/main/java/io/ballerina/persist/cmd/Generate.java

```diff
@@ -49,6 +49,8 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+//import static io.ballerina.persist.nodegenerator.BalSyntaxTreeGenerator.generateRelations;
```

Comment:
```suggestion
```

Reply:
updated in [fe79bc0](https://github.com/ballerina-platform/persist-tools/pull/23/commits/fe79bc066762c9cbdb981fbfb081b6efea3af0d4)